runtime/cmake/caches/common.cmake

```diff
@@ -170,6 +172,7 @@ set(J9VM_INTERP_NEW_HEADER_SHAPE ON CACHE BOOL "")
 set(J9VM_INTERP_PROFILING_BYTECODES ON CACHE BOOL "")
 set(J9VM_INTERP_SIG_QUIT_THREAD ON CACHE BOOL "")
 set(J9VM_INTERP_SIG_QUIT_THREAD_USES_SEMAPHORES ON CACHE BOOL "")
+set(J9VM_TWO_PASS_EXCLUSIVE ON CACHE BOOL "")
```

Comment:
two pass exclusive isn't enabled for all specs in the UMA world.  Should this be gated with per-platform checks in the same way?  Similar concern about the other two options

Reply:
I believe it is currently enabled for all specs which we support under cmake. The thought being that we can then factor it out when we bring a new platform online which does not support it

Reply:
Given some of the issues so far with turning on cmake, I'm not a fan of this approach.  It means having to validate every flag on every spec over and over again.  If we handle the flags once, we'll be confident it's set correctly when support for a new spec is added

Reply:
Moved out into the individual configs